src/Protocol/Protocol_1_12.cpp

```diff
@@ -1296,6 +1296,26 @@ void cProtocol_1_12::SendUpdateBlockEntity(cBlockEntity & a_BlockEntity)
 
 
 
+void cProtocol_1_12::HandlePacketCraftingBookData(cByteBuffer & a_ByteBuffer)
+{
+	a_ByteBuffer.SkipRead(a_ByteBuffer.GetReadableSpace() - 1);
+	m_Client->GetPlayer()->SendMessageInfo("The feature crafing book (green) is not implemented.");
```

Comment:
I'm not sure about the English of this. I myself would have written `The green crafting book feature is not implemented yet.`

Need some native English speaker :)